client/api_group.go

```diff
@@ -568,3 +570,106 @@ func (c *client) ListGroupsByOwner(ctx context.Context, opts types.GroupsOwnerPa
 
 	return groups, nil
 }
+
+type GfSpListGroupsByGroupIDsResponse map[uint64]*types.GroupMeta
+
+type GroupEntry struct {
```

Comment:
add comment

Reply:
No need to expose these fields to the user, please start with lowercase letters.

Reply:
fixed